acestep/api/http/release_task_request_builder.py

```
@@ -36,6 +36,17 @@ def build_generate_music_request(
     if track_classes is not None and isinstance(track_classes, str):
         track_classes = [track_classes]
 
+    guidance_params = parser.get("guidance_params")
+    if isinstance(guidance_params, str):
+        import json as _json
+        try:
+            parsed_guidance = _json.loads(guidance_params)
+            guidance_params = parsed_guidance if isinstance(parsed_guidance, dict) else None
+        except Exception:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)